tests/e2e/lib/restore.go

```diff
@@ -22,6 +23,9 @@ func CreateRestoreFromBackup(ocClient client.Client, veleroNamespace, backupName
 			BackupName: backupName,
 		},
 	}
+	if restorePVs {
+		restore.Spec.RestorePVs = pointer.Bool(true)
```

Comment:
maintainers, what this option does with the restore?

Reply:
If I'm remembering correctly, this the restore equivalent to snapshotVolumes -- @shubham-pampattiwar is this correct?
Assuming I'm not misremembering, this means, similar to above, if it's true then we restore the PVC content from snapshot if it is a volume that was backed up via snapshot.

Reply:
RestorePVs specifies whether to restore all included PVs from snapshot

Reply:
+1 @sseago

Reply:
@shubham-pampattiwar It's confusing that the restore and backup flags aren't named similarly, but they are essentially equilalent flags, but one for backup, the other for restore.

Reply:
> The snapshot can be cloud provider based or csi snapshot. Both are applicable.

Same with SnapshotVolumes. There was a fix a while back to make this flag do the same thing for CSI and native snapshots.

Reply:
yes since velero 1.12 I think, initially these 2 flags were only applicable for cloud provider based snapshots.